extension_helpers/_utils.py

```diff
@@ -77,7 +78,7 @@ def walk_skip_hidden(top, onerror=None, followlinks=False):
         yield root, dirs, files
 
 
-def write_if_different(filename, data):
```

Comment:
This is the breaking change I was talking about. Can you please revert it ? Beyond that, happy to approve this PR.

Reply:
Done and pushed